eureka_ml_insights/data_utils/spatial_utils.py

```diff
@@ -198,181 +204,65 @@ def extract_answer_from_text_map_and_maze(model_output_raw, options):
 
     # get dict of options from options string
     options_dict = {x.split(".")[0].strip().lower():x.split(".")[1].strip().lower() for x in options}
-
-
-    model_output_parsed_letter = ""
-    model_output_parsed = ""
             
     answers = [v for k, v in options_dict.items()]
     answers_pattern = rf"\b({'|'.join(answers)})\b"
```

Comment:
Could you please move letters and letters_pattern here for readability and consistency?